File: src/api/LMSEntities/Models/LibraryAsset_Author.cs
namespace LMSEntities.Models
{
    public class LibraryAssetAuthor
    {
        // public int Id { get; set; }
        public int LibrayAssetId { get; set; }
        public LibraryAsset LibraryAsset { get; set; }
        public int AuthorId { get; set; }
        public Author Author { get; set; }
        public byte Order { get; set; }
    }
}

File: src/spa/src/app/dto/models/library-asset-author.ts
/* tslint:disable */
/* eslint-disable */
import { Author } from './author';
import { LibraryAsset } from './library-asset';
export interface LibraryAssetAuthor {
  author?: Author;
  authorId?: number;
  libraryAsset?: LibraryAsset;
  librayAssetId?: number;
  order?: number;
}
